test/integration/overload_integration_test.cc

```diff
@@ -0,0 +1,93 @@
+#include "test/integration/http_protocol_integration.h"
+
+#include "absl/strings/str_cat.h"
+#include "absl/strings/substitute.h"
+
+namespace Envoy {
+
+class OverloadIntegrationTest : public HttpProtocolIntegrationTest {
+protected:
+  OverloadIntegrationTest()
+      : injected_resource_filename_(TestEnvironment::temporaryPath("injected_resource")),
+        file_updater_(injected_resource_filename_) {}
+
+  void initialize() override {
+    config_helper_.addConfigModifier([this](envoy::config::bootstrap::v2::Bootstrap& bootstrap) {
+      const std::string overload_config = absl::Substitute(R"EOF(
```

Comment:
We tend to prefer YAML to text proto in test literal blocks like this; it's more readable and familiar to non-Googlers. Most test extended strings with these fragments should look like this if you want some examples.

Reply:
Done